tests/dummy/app/templates/application.hbs

```diff
@@ -2,4 +2,4 @@ Hello, world!
 
 <button class="button-default">Button Default</button>
 
-{{outlet}}
+{{outlet}}
```

Comment:
new end of line

Reply:
This was done by the automated update tool, may be worth committing a change back to fix it